projects/go-style-guide/src/app/features/ui-kit/components/modal-docs/modal-docs.component.ts

```diff
@@ -67,10 +72,26 @@ export class ModalDocsComponent {
 
   ex_ModalDocsHtml: string = `<go-button (handleClick)="openModal()">Click Me</go-button>`;
 
+  ex_ModalDocsOpenLgModal: string = `
+  this.goModalService.openModal(ModalTestComponent, { modalTitle: 'LG Modal (Default)', modalSize: 'lg', content: 'This is a lg modal' });
```

Comment:
we don't need to pass in a modalSize here correct because it defaults to `'lg'`?

Reply:
correct that it's not mandatory to pass, but I'm passing it here just in case we ever change the default. Less to worry about updating later, however if you feel it would make more sense to remove the `modalSize` here then I can.

Reply:
correct that it's not mandatory to pass, but I'm passing it here just in case we ever change the default. Less to worry about updating later, however if you feel it would make more sense to remove the `modalSize` here then I can.